packages/contracts-bedrock/snapshots/state-diff/Kontrol-Deploy.json

```diff
@@ -1954,7 +1954,7 @@
           "newValue": "0x0000000000000000000000000000000000000000000000000000000000000000",
           "previousValue": "0x0000000000000000000000000000000000000000000000000000000000000000",
           "reverted": false,
-          "slot": "0x0000000000000000000000000000000000000000000000000000000000000038"
+          "slot": "0x0000000000000000000000000000000000000000000000000000000000000039"
```

Comment:
Confirming this snapshot diff is justified: The account here, 0x9568d36E291c2C4c34fa5593fcE73715abEf6F9c, is the DeployConfig contract. A new variable was added to that contract's storage—`bool public enableGovernance`—so it makes sense a slot shifted back by 1